specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4861,6 +4861,95 @@
         "authenticationType"
       ]
     },
+    "SapTable": {
+      "x-ms-discriminator-value": "SapOpenHub",
+      "description": "SAP Table Linked Service.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/LinkedService"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Properties specific to this linked service type.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SapTableLinkedServiceTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "SapTableLinkedServiceTypeProperties": {
+      "description": "Properties specific to this linked service type.",
+      "properties": {
+        "server": {
+          "type": "object",
+          "description": "Host name of the SAP instance where the table is located. Type: string (or Expression with resultType string)."
+        },
+        "systemNumber": {
+          "type": "object",
+          "description": "System number of the SAP system where the table is located. (Usually a two-digit decimal number represented as a string.) Type: string (or Expression with resultType string)."
+        },
+        "clientId": {
+          "type": "object",
+          "description": "Client ID of the client on the SAP system where the table is located. (Usually a three-digit decimal number represented as a string) Type: string (or Expression with resultType string)."
+        },
+        "language": {
+          "type": "object",
+          "description": "Language of the SAP system where the table is located. The default value is EN. Type: string (or Expression with resultType string)."
+        },
+        "systemId": {
+          "type": "object",
+          "description": "SystemID of the SAP system where the table is located. Type: string (or Expression with resultType string)."
+        },
+        "userName": {
+          "type": "object",
+          "description": "Username to access the SAP server where the table is located. Type: string (or Expression with resultType string)."
+        },
+        "password": {
```

Comment:
Friendly reminder: please ensure that the server never returns secrets on get nor logs requests.

Reply:
This is already taken care of - we encrypt all secrets, and our service returns only encrypted values.